errors.go

```diff
@@ -0,0 +1,11 @@
+package gdrive
+
+import (
+	"github.com/beyondstorage/go-storage/v4/services"
+)
+
+var (
+	ErrRateLimitExceeded  = services.NewErrorCode("Rate limit exceeded")
+	ErrBackEnd            = services.NewErrorCode("Backend Error")
```

Comment:
Please use `ErrServiceInternal` instead.

Ref:

- https://github.com/beyondstorage/specs/blob/master/rfcs/127-add-serviceinternel-and-requestthrottled-errors.md
- https://github.com/beyondstorage/go-storage/issues/612

Reply:
Is this proposal has been implemented? I just can't find it in `go-storage/services/error.go` Can we pass the tests if I use it?

Reply:
> Is this proposal has been implemented? I just can't find it in `go-storage/services/error.go` Can we pass the tests if I use it?

ping @xxchan to get the plan for implementations of GSP-127.

As GSP-127 is not implemented, return error like `erros.New("xxx")` is OK to me. It's better than define a public error too early.

Reply:
How about just simply use these unimplemented errors, we can left the failed tests and merge the PR when GSP-127 is done.

Reply:
OK.